app/src/main/java/org/gdg/frisbee/android/common/GdgNavDrawerActivity.java

```diff
@@ -75,7 +79,25 @@
     ImageView mDrawerImage;
     ImageView mDrawerUserPicture;
     private ActionBarDrawerToggle mDrawerToggle;
-    private MenuItem drawerItemToNavigateAfterSignIn = null;
+
+    int drawerItemIdToNavigateAfterSignIn = INVALID_ITEM_ID;
```

Comment:
This was MenuItem before. I want this to survive orientation/config changes. Since MenuItem is not Parcelable, I cannot put it in Bundle. 
Maybe orientation change related problems with Sign In could be put in a different PR. Sorry about that.